packages/web-component/src/lib/helpers/templates.ts

```diff
@@ -104,6 +104,20 @@ const replaceHrefByDataType = (
   });
 };
 
+const enableDisableInputs = (
+  baseEle: DocumentFragment,
+  formData: Record<string, string>,
+) => {
+  Object.keys(formData).forEach((name) => {
+    const eles = baseEle.querySelectorAll(`[name="${name}"]`);
+    eles.forEach((ele) => {
+      Object.keys(formData[name]).forEach((attr) => {
+        ele.setAttribute(attr, formData[name][attr]);
+      });
+    });
+  });
```

Comment:
this could be a bit risky. setting any attribute given by the user inputs can cause a security issue here. i think we need to force only what we support (disabled)

Reply:
@Bars92 Added allow list.

Reply:
@tomerlichtash where?

Reply:
Weird github preview. Here: https://github.com/descope/descope-js/pull/520/commits/d1423d664618017a50ed384fa8462d27f1784a78#diff-14e4f9a9bb67ac1b8961131ee345a4b2eeec8792f6e6a081d0abb1c32d7dba3bR8